.gitattributes

```diff
@@ -1,2 +1,3 @@
-spec/fixtures/vcr_cassettes -diff
+*.rb diff=ruby
+spec/fixtures/vcr_cassettes -diff -merge
```

Comment:
I hope this works - it would be great to get the vcr cassettes out of the github diffs.